comictaggerlib/autotagmatchwindow.py

```diff
@@ -229,8 +229,8 @@ def reject(self) -> None:
     def save_match(self) -> None:
         match = self.current_match()
         ca = ComicArchive(self.current_match_set.original_path)
+        md, success = self.parent().overlay_ca_read_style(self.load_data_styles, ca)
```

Comment:
`success` is never used, either remove the return value from `overlay_ca_read_style` or do something with it here